app/models/concerns/auction_scopes.rb

```diff
@@ -11,13 +11,13 @@ module AuctionScopes
     scope :default_purchase_card, -> { where(purchase_card: 0) }
     scope :delivered, -> { where.not(delivery_url: [nil, '']) }
     scope :delivery_due_at_expired, -> { where('delivery_due_at < ?', Time.current) }
+    scope :delivery_needed, -> { where(status: statuses['pending_delivery']) }
```

Comment:
I was trying to keep the naming similar to how we have it on the needs attention page but I see your point - will update!